webpack.config.js

```diff
@@ -50,6 +50,7 @@ module.exports = (env) => {
       bundlesLandingPage: 'pages/bundles-landing/bundlesLanding.jsx',
       monthlyContributionsPage: 'pages/monthly-contributions/monthlyContributions.jsx',
       monthlyContributionsThankyouPage: 'pages/monthly-contributions-thankyou/monthlyContributionsThankyou.jsx',
+      monthlyContributionsExistingPage: 'pages/monthly-contributions-existing/monthlyContributionsExisting.jsx',
```

Comment:
Does this mean having to use [extra](https://webpack.js.org/plugins/commons-chunk-plugin/#commons-chunk-for-entries) `script` tags?